src/background/passes.js

```diff
@@ -123,7 +123,7 @@ export class Passes extends Component {
       version = null;
     }
 
-    if (version !== "12" || details.previousVersion === version) {
+    if (details.previousVersion > "11") {
```

Comment:
This is a bit shady string to int conversion. Maybe make it explicit?

Reply:
The versions are actually strings. What do you suggest?

Reply:
Well, right now you're implicitly casting both to numbers, so maybe just `parseInt(details.previousVersion) > 11`?